README.md

```diff
@@ -74,5 +74,20 @@ Example: feat(news-feed): add news feed feature on landing page
 git push origin master -f
 ```
 
+## Contribution
+
+<!-- Please make sure to read the [Contributing Guide](https://github.com/shoonyatech/frontend.social/blob/master/.github/CONTRIBUTING.md) before making a pull request.  -->
+
+Thank you to all the people who already contributed to frontend.social!
+
+<a href="https://github.com/shoonyatech/frontend.social/graphs/contributors"><img src="./.github/contributors.svg" /></a>
+
+## Stay in Touch
+
+> - Twitter - @FrontendSocial
```

Comment:
Show bullet instead of > - at the start of the line and do not show link as text but as a hyperlink
As already mentioned, please take reference of how it is done in the vuejs github page
![image](https://user-images.githubusercontent.com/195256/94986994-db695c00-0580-11eb-966d-295fd4926c4d.png)
https://github.com/vuejs/vue

Reply:
Yeah got it !!